app/views/index.njk

```diff
@@ -1,89 +1,98 @@
+    <p><strong>The result?</strong> You can develop and launch projects faster and more efficiently to improve the lives of patients, clinicians and NHS workers, ensuring public money doesn’t get wasted through duplicating efforts and creating a documented audit trail of your governance journey.</p>
+    <p>This manual is designed to help all technical and/or project professionals get to the right information faster through guided navigation. Simply select what you want to achieve either from the homepage or using the side navigation and let the manual take you to the relevant content.</p>
     <p>Where available, we will also recommend additional information or resources which might be useful to you and your job.</p>
-    <p>If you have any content you would like to add or update within the eBook, please email your request to <a href="mailto:trg.mailbox@nhs.net">trg.mailbox@nhs.net</a>.
-  </div>
-</div>
+    <p>If you have any content you would like to add or update within the manual, please email your request to <a href="mailto:trg.mailbox@nhs.net">trg.mailbox@nhs.net</a>.
 
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-three-quarters">
-    <h2 id="what-do-you-want-to-do-today">What do you want to do today?</h2>
-  </div>
-</div>
 
-<div class="nhsuk-grid-row nhsuk-promo-group">
-  <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
-    <div class="nhsuk-promo nhsuk-promo--small">
-      <a class="nhsuk-promo__link-wrapper" href="/#page">
-        <div class="nhsuk-promo__content">
-          <h3 class="nhsuk-promo__heading">I want to find a specific principle, strategy, policy or standard</h3>
-          <p class="nhsuk-promo__description">Helping me deliver projects faster. Avoid wastage. Creating consistency.</p>
-        </div>
-      </a>
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-three-quarters">
+      <h2 id="what-do-you-want-to-do-today">What do you want to do today?</h2>
     </div>
   </div>
-  <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
-    <div class="nhsuk-promo nhsuk-promo--small">
-      <a class="nhsuk-promo__link-wrapper" href="/#page-1">
-        <div class="nhsuk-promo__content">
-          <h3 class="nhsuk-promo__heading">I know something exists. I want to find out if it is suitable for me to re-use it</h3>
-          <p class="nhsuk-promo__description">Saving me development time and effort. Helping create a joined-up organisation.</p>
+
+    <div class="nhsuk-grid-row nhsuk-promo-group">
+      <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
+        <div class="nhsuk-promo nhsuk-promo--small">
+          <a class="nhsuk-promo__link-wrapper" href="/#page">
```

Comment:
Are we still waiting for pages for these Promo links?

Reply:
Yes that's right, I have let the content authors know we are still waiting for these changes, they will be updated in the next round of content changes. I will leave this open for reference.